marklogic-data-hub/src/main/resources/ml-modules/transforms/run-flow.xqy

```diff
@@ -0,0 +1,25 @@
+xquery version "1.0-ml";
+
+module namespace runFlow = "http://marklogic.com/rest-api/transform/run-flow";
+
+import module namespace flow = "http://marklogic.com/hub-in-a-box/flow-lib"
+at "/com.marklogic.hub/lib/flow-lib.xqy";
+
+declare function runFlow:transform(
+  $context as map:map,
+  $params as map:map,
+  $content as document-node()
+  ) as document-node()
+{
+  let $entityName := map:get($params, 'entity-name')
+  let $flowName := map:get($params, 'flow-name')
+  
+  let $flow := flow:get-flow($entityName,$flowName,())
```

Comment:
close. you still need to pass in input here. Just don't make it a user supplied param.